examples/k8s/deployment.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: cert-manager.io/v1alpha2
 kind: Issuer
 metadata:
-  name: argoslower-selfsigned
+  name: selfsigned-argoslower
```

Comment:
Sneaking William's suggestion in here to avoid having to add namespace annotation `issuer-allowlist`.